integration/go.mod

```diff
@@ -13,14 +13,26 @@ require (
 )
 
 require (
```

Comment:
Deps were messed up by the previous commit

Reply:
Do you understand where are all the new indirect dependencies from?

Reply:
From the Tigris handler. That's not a problem in integration tests